code/game/objects/items/toys.dm

```diff
@@ -603,10 +613,16 @@
 
 
 /obj/item/toy/plushie/proc/explosive_betrayal(obj/item/grenade/grenade_callback)
+	var/grenade_inside = FALSE //Any grenade, even non-explosive, will destroy the plushie.
+	if(grenade)
+		grenade_inside = TRUE
 	grenade_callback.prime()
+	if(grenade_inside && !QDELETED(src))
+		qdel(src)
 
 /obj/item/toy/plushie/Destroy()
 	QDEL_NULL(grenade)
+	QDEL_NULL(cursed_plushie_victim)
```

Comment:
Are you sure you want to qdel the victim, or would it be better to forcemove them out instead?

Reply:
Allows the wizard to use it more offensively, similar to how you can destroy a statue created by flesh to stone (same price / cooldown) to dust someone.

Reply:
I feel it gives the wizard the option to get rid of someone if they _really_ want to put the effort in to do so.